### PR TITLE
Change invariant in paths.js to a FatalError

### PR DESCRIPTION
**Pull request by trueadm**
Release notes: none

I can't reproduce this invariant without using the UFI bundle, so unfortunately there are no tests attached. This unblocks the React reconciler work. Also, interestingly this causes memory to explode (I guess we're now evaluating far more than before since this invariant was blocking that from happening).

### CODE REVIEW

File: src/methods/call.js [hunk 1]
```diff
@@ -432,6 +432,8 @@ export function OrdinaryCallEvaluateBody(
             invariant(c instanceof Value || c instanceof AbruptCompletion);
             return c;
           }
+        } catch (e) {
+          throw e;
```
Comment (gaearon) [inline src/methods/call.js]: This doesn't mean anything, does it?
A rethrow catch clause is equivalent to no catch clause.

Comment (trueadm) [reply src/methods/call.js]: Opps, that was left in from debugging, I'll remove.

**Comment by hermanventer** [comment]
I much prefer having an invariant here. This is not a limitation of prepack but a failure in its implementation. If you can get me any kind of repro I can try to fix the failure.

**Comment by trueadm** [comment]
Closing in favour of a proper fix.

**Pull request closed without merging**